### ISSUE TITLE
What happened to :delegated, :cached and :ro?

### ISSUE THREAD

**Issue by osteel** [docker/for-mac#5402]
Not technically a bug but I can't seem to find the answer anywhere: the official documentation used to reference `:delegated`, `:cached` and `:ro` flags mostly to help with performance on macOS but they are now nowhere to be found.

Have they been made redundant with a recent version?

**Comment by stephen-turner** [comment]
Yes, they don't apply with the grpcfuse back end.

**Comment by osteel** [comment]
@stephen-turner thank you for replying even though that's not technically the right place for this kind of questions.

I would just add that if that is indeed the case, and as gRPC-FUSE is currently flagged as an experimental feature, removing these flags from the documentation seems a bit premature, even if gRPC-FUSE is now enabled by default.

**Comment by stephen-turner** [comment]
You're right. The confusion is because gRPC-FUSE is not really experimental. It just got left in the experimental tab accidentally from when it was experimental. The next release (this week) will move it into the General tab.

In some ways we would have liked to leave the osxfs documentation up while osxfs was still available as an option, but we found it was causing a lot of confusion and thought it was better to remove it early.

**Comment by osteel** [comment]
@stephen-turner thank you for confirming.
